TestAssets/TestProjects/DesktopUsingPackageWithSatellites/DesktopUsingPackageWithSatellites.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
I'll add the coverage. 

The diff shown in #1132 also changes from netstandard1.4 to net46. Are you sure that you had `<TargetFramework>net46</TargetsFramework>` working and only `<TargetFrameworks>net46[;...]</TargetFrameworks>` failing? This very test repros the issue without using `TargetFrameworks`.